js/src/data/index.js

```diff
@@ -40,6 +41,18 @@ apiFetch.use(
 					return errorInfo;
 				} )
 				.then( ( errorInfo ) => {
+					const { code } = errorInfo;
+
+					if ( code === 'JETPACK_DISCONNECTED' ) {
+						getHistory().replace( getReconnectWPComAccountUrl() );
+					} else if ( code === 'GOOGLE_DISCONNECTED' ) {
+						getHistory().replace( getReconnectGoogleAccountUrl() );
+					}
```

Comment:
💅 WDYT of saving `JETPACK_DISCONNECTED` and `GOOGLE_DISSCONECTED`as constants?

Reply:
Added in https://github.com/woocommerce/google-listings-and-ads/pull/1385/commits/9eb735608b1945a2d546d9421cdcf1d426211923.